botx/models/entities.py

```diff
@@ -1,11 +1,34 @@
-"""Pydantic models for mentions."""
+"""Forward entities that can be received in message."""
```

Comment:
`Entities that can be received in message`?

Reply:
that field has name `entities`